app/views/nsm/steps/view_claim/show.html.erb

```diff
@@ -21,6 +21,11 @@
         <li class="moj-sub-navigation__item">
           <%= link_to t('.tabs.claimed_costs'), nsm_steps_view_claim_path(@claim, section: 'claimed_costs'), class: 'moj-sub-navigation__link', 'aria-current': (@section == :claimed_costs ? "page" : nil) %>
         </li>
+        <% if @claim.status == 'part_grant' %>
```

Comment:
the statuses have enums you could use but either way

```suggestion
        <% if @claim.part_grant? %>
```

Reply:
let me look at this as I tried it and it didn't work... :(